src/main/java/com/stories/service/StoriesServiceImpl.java

```diff
@@ -169,6 +169,48 @@ else if(mensaggeDinamicValidation[2] == "CONFLICT") {
             }
 		}
 	}
+	
+	@Override
+	public TasksDomain updateTaskById(TasksDomain task, String id, String _id) throws Exception {
+		if(storiesRepository.existsById(id)) {
+			task.set_id(_id);
+			TaskModel taskModel = storiesCustomRepository.getTaskById(id, _id).getUniqueMappedResult();
+			if(taskModel == null) {
+    			throw new EntityNotFoundException("Task not found", "/stories/" + id + "/tasks/" + _id);
+    		}
+			if(!StringUtils.isEmpty(task.getAssignee())) {
+				if(!usersRepository.existsById(task.getAssignee())) {
+					throw new EntityNotFoundException("User assignee id does not exist", HttpStatus.CONFLICT, "/stories/" + id + "/tasks/" + _id);
+				}
+			}
+			if(StringUtils.isEmpty(task.getName())) {
+				throw new EntityNotFoundException("name is a required field", HttpStatus.BAD_REQUEST, "/stories/" + id + "/tasks/" + _id);
+			}
+			storyModel = storiesRepository.findById(id).get();
+			List<TaskModel> FinalTasksList = new ArrayList<>();
```

Comment:
Les´t change the name of this variable from FinalTasksList to updatedTaks please.